### DISCUSSION TITLE
Nested data providers with filter not working anymore

### DISCUSSION THREAD

**Discussion by mvhaen** [Budibase/budibase#7648] [Help] [answered]
For an internal dashboard application we wanted to show a bunch of pie charts on 1 page, but putting a single data provider in place for all these charts resulted in timeouts.

We then found a workaround by creating 1 query that returned all the results and then creating child data providers that filtered data from the main one as seen below.

<img width="294" alt="image" src="https://user-images.githubusercontent.com/157020/188501509-11b7f402-ce78-4f0c-a79f-8654cf58cc69.png">

We now wanted to expand this page with some extra graphs. For some reason every new nested data provider starts returning zero results when we add a filter (expect if we add a filter on is empty then we get all results). 

I've started a new page + query from scratch and it shows the same results. The pie charts we set up before still work, when I duplicate the provider and the chart under it, it also works but it breaks when I start changing the filter (even if I change it to a filter value which I know returns results on another chart).

Did something change here in a recent release?

**Comment by melohagan** [comment]
Hey @mvhaen 

Could you share some screenshots of the filters being used please? The app export may also be helpful if possible.

**Comment by mvhaen** [comment, marked as answer]
The bug already got fixed. See this issue which also has more details and an export: https://github.com/Budibase/budibase/issues/7691

**Reply by melohagan** [reply]
Brilliant, thanks for getting back!
